templates/infrastructure/variables.tf

```diff
@@ -41,6 +41,12 @@ variable "tags" {
   # }
 }
 
+variable "kube_version" {
+  description = "Kubernetes version"
+  default = "4.9.21_openshift"
```

Comment:
pls check the latest iks/openshift version and use it as default for now

Reply:
Don't set default itself cluster will take which ever is default at that time

Reply:
Can we move this also to v ariables files where user can provide value.
If we keep some default value after few days the version will be unsupported